tsconfig.json

```diff
@@ -1,43 +1,20 @@
 {
-  "compileOnSave": false,
   "compilerOptions": {
-    "target": "es2018",
-    "module": "esnext",
-    "sourceMap": true,
-    "allowSyntheticDefaultImports": true,
-    "downlevelIteration": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "module": "esnext",
     "moduleResolution": "node",
-    "allowJs": true,
-    "baseUrl": "src",
     "skipLibCheck": true,
-    "strict": true,
-    "lib": [
-      "es2018",
-      "dom"
-    ],
-    "types": [ "i18next", "i18next-xhr-backend" ],
-    "typeRoots": [
-      "node_modules/@types",
-      "src/typings"
-    ]
+    "target": "ES2017",
```

Comment:
can we target higher than 2017? I was targeting ES2018 before, we can probably go with at least ES2018 and perhaps even ES2020 for Au2, what do you think?

Reply:
I'll try. From my previous experience, es2020 did not work with Aurelia1 decorators, not sure about AU2